build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -121,17 +121,13 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
                     {
                         "win.x86.version",
                         "win.x64.version",
-                        "win.arm64.version",
```

Comment:
My mistake, arm64 just had an error in the build I looked at--the leg is still running. Reverting this line.